src/components/ViewAdmin.vue

```diff
@@ -255,6 +259,15 @@ export default {
 				this.success = false
 			}, 3000)
 		},
+		async onRescan() {
+			this.loading = true
+			await axios.get(generateUrl('/apps/recognize/admin/recrawl'))
+			this.loading = false
+			this.success = true
+			setTimeout(() => {
```

Comment:
Might be nicer to catch any thrown error around the axios call and in case a timeout is needed, set that as an axios option. That way also other request errors are catched.

Reply:
The admin view is currently rather hacky in general, I think I'll polish it in a new PR.